### PR TITLE
replaced one screenshot

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1766

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: Office/OfficeExperts/microsoft-forms/no-email-notifications-for-responses-to-group-forms.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ You might not receive email notifications for responses that you submit to a Gro
 
    ![Selecting your group](./media/no-email-notifications-for-responses-to-group-forms/select-group.png)
 
-4. On the right side, you see the number of members in the group. Select that number, and then it opens the group email page in Outlook.
+4. On the right side, you see the number of members in the group. Select that number, and then it opens the group email page in Outlook on the Web.
```
Comment (JohanFreelancer9) [inline Office/OfficeExperts/microsoft-forms/no-email-notifications-for-responses-to-group-forms.md:35]:
```suggestion
4. On the right side, you see the number of members in the group. Select that number, and then it opens the group email page in Outlook on the web.
```

File: Office/OfficeExperts/microsoft-forms/no-email-notifications-for-responses-to-group-forms.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ You might not receive email notifications for responses that you submit to a Gro
...
 
    ![Selecting the number of members in the top-right](./media/no-email-notifications-for-responses-to-group-forms/number-of-members.png)
 
-5. In Outlook web app, select the ellipsis button (...) next to **Send email**, and in the drop-down box select **Edit group**.
+5. In Outlook on the Web, select **Edit** button above the group name.
```
Comment (JohanFreelancer9) [inline Office/OfficeExperts/microsoft-forms/no-email-notifications-for-responses-to-group-forms.md:39]:
```suggestion
5. In Outlook on the web, select the **Edit** button above the group name.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**